src/lib/philips.ts

```diff
@@ -94,7 +94,44 @@ const philipsModernExtend = {
             result.exposes.push(...exposeEndpoints(e.enum("effect", ea.SET, effects), args.endpointNames));
         }
 
-        const customCluster = m.deviceAddCustomCluster("manuSpecificPhilips3", {
+        const customCluster = m.deviceAddCustomCluster("manuSpecificPhilips", {
+            ID: 0xfc00,
+            manufacturerCode: Zcl.ManufacturerCode.SIGNIFY_NETHERLANDS_B_V,
+            attributes: {
+                config: {ID: 49, type: Zcl.DataType.BITMAP16},
+            },
+            commands: {},
+            commandsResponse: {
+                hueNotification: {
+                    ID: 0,
+                    parameters: [
+                        {name: 'button', type: Zcl.DataType.UINT8},
+                        {name: 'unknown1', type: Zcl.DataType.UINT24},
+                        {name: 'type', type: Zcl.DataType.UINT8},
+                        {name: 'unknown2', type: Zcl.DataType.UINT8},
+                        {name: 'time', type: Zcl.DataType.UINT8},
+                        {name: 'unknown2', type: Zcl.DataType.UINT8},
+                    ],
+                },
+            },
+        });
+
+        const customCluster2 = m.deviceAddCustomCluster("manuSpecificPhilips2", {
```

Comment:
Suggestion: Name these consts after the cluster id (`customClusterFC03`, or such). That way, we never have to renumber them, if we find any new clusters.

Reply:
I don't know if it's acceptable to rename the clusters themselves. Maybe it's possible to add an alias name, and deprecate the `manuSpecificPhilipsX` names over time?

Reply:
See latest commit

Reply:
The cluster namings will be part of a larger re-factor. I have aired on the side of a lift and drop for now, with segmented changes in subsequent PRs to break up the monolithic nature of what we will need to do